src/main/target/BEEROTORF4/target.h

```diff
@@ -70,9 +70,7 @@
 #define SDCARD_SPI_FULL_SPEED_CLOCK_DIVIDER     4 // 21MHz
 
 #define SDCARD_DMA_CHANNEL_TX               DMA1_Stream4
-#define SDCARD_DMA_CHANNEL_TX_COMPLETE_FLAG DMA_FLAG_TCIF4
-#define SDCARD_DMA_CLK                      RCC_AHB1Periph_DMA1
-#define SDCARD_DMA_CHANNEL                  DMA_Channel_0
+#define SDCARD_DMA_CHANNEL                  0
```

Comment:
@blckmn: This seems to be wedging the BEEROTORF4 board (tested b73ffbb59230d1f82d20c02b8ed080c4adae5197 which is working fine). Boot blinks happen, USB device is initialised, but LED0 stays on. Any idea what could be wrong here?